app/src/main/java/cz/martykan/forecastie/weatherapi/WeatherStorage.java

```diff
@@ -0,0 +1,146 @@
+package cz.martykan.forecastie.weatherapi;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+import android.util.Log;
+
+import androidx.annotation.Nullable;
+
+import org.json.JSONException;
+
+import java.util.List;
+
+import cz.martykan.forecastie.Constants;
+import cz.martykan.forecastie.models.Weather;
+import cz.martykan.forecastie.weatherapi.owm.OpenWeatherMapJsonParser;
+
+public class WeatherStorage {
+    protected SharedPreferences sharedPreferences;
+
+    public WeatherStorage(Context context) {
+        this.sharedPreferences = PreferenceManager.getDefaultSharedPreferences(context);
+    }
+
+    public int getCityId()
+    {
```

Comment:
The curly braces in the pull request aren't consistent. Put them in the same line, like this:
```java
public int getCityId() {
```

Reply:
absolutely. we don't have a formally-defined coding style, but i agree we should be consistent

Reply:
Fixed in 3629f50fbee9acd5ff180c741355bb80471d1f6b